.circleci/config.yml

```diff
@@ -19,3 +32,21 @@ jobs:
       - run:
           command: ./.circleci/run.sh run-make
           name: Make the release target
+      - persist_to_workspace:
+          root: ~/dlang.org
+          paths:
+            - web
+  deploy:
+    working_directory: ~/dlang.org
+    docker:
+      - image: circleci/node
+    steps:
+      - attach_workspace:
+          # Must be absolute path or relative path from working_directory
+          at: ~/dlang.org
+
+      - run:
+          command: |
+            npm i netlify-cli
+            npx netlify deploy --site "$NETLIFY_SITE_ID" --auth "$NETLIFY_ACCESS_TOKEN" --prod --dir=web
```

Comment:
@zombinedev FYI: we could also run this on all builds without the `--prod` flag and then get a deployment preview. However, as we already have DAutoTest and there's not easy way to get the link back from the logs to GH, I'm not sure whether this is useful.

Reply:
We can save the output of `netlify deploy` to a file (or just stdin it) and grep for the URL. Then perhaps we can use @dlang-bot to post the link to the PR?